### ISSUE TITLE
Add support for Langfuse

### ISSUE THREAD

**Issue by homanp** [superagent-ai/slack-bot#3]
A user should be able to rate the slack-bot using the thumbs up/down emoji on the both. 

These signals should best to langfuse.

**Comment by homanp** [comment]
@elisalimli I want to add support for observability. I'm thinking I want to use OpenLLMetry for this. When a user presses 👍 👎  this should be added to langfuse et al. What do you think?

**Comment by elisalimli** [comment]
> @elisalimli I want to add support for observability. I'm thinking I want to use OpenLLMetry for this. When a user presses 👍 👎 this should be added to langfuse et al. What do you think?

@homanp Yes! But why we would use OpenLLMetry? We have the just input, output (from superagent API call) and the user feedback. Can't we do all of these 3 things using Langfuse?
